plugins/rh-rbac-backend/src/service/permission-policy.ts

```diff
@@ -26,14 +31,41 @@ e = some(where (p.eft == allow)) && !some(where (p.eft == deny))
 m = r.sub == p.sub && r.obj == p.obj && r.act == p.act
 `;
 
+const useAdmins = (admins: Config[], enf: Enforcer) => {
+  admins.flatMap(async localConfig => {
+    const name = localConfig.getString('name');
+    const adminPermission = [name, 'permission-entity', 'read', 'allow'];
```

Comment:
Right now, I just added the `permission-entity` `read` Since we only have the one route exposed. I figured whenever we begin work on adding the CRUD operations, I could go in and add all of the permissions at once. However, I can add them now if you would prefer it.

Reply:
Yes. When I think about it, we may not need this many permissions, only read and write. Also, the current config does not allow us to configure them separately, so it is OK to add both now.